frontend/public/components/utils/request-size-input.tsx

```diff
@@ -49,6 +49,11 @@ export class RequestSizeInput extends React.Component<RequestSizeInputProps> {
             ariaLabel={`Number of ${this.props.dropdownUnits[this.props.defaultRequestSizeUnit]}`}
           />
         </div>
+        {helpText && (
+          <p className="help-block" id="request-size-help">
```

Comment:
This won't be unique if the component appears on the page more than once.

Reply:
The `aria-describedby` attribute also won't match if a different value was passed into the component.

Reply:
I removed the `id`, 

regarding `ResourceLimitField`, its under a `FormGroup` that already takes care of the `helpText`.
the prop here is used for implementations not using a parent `FormGroup`

Reply:
> I removed the `id`,

You can't remove `id`. It's needed for `aria-describedby` to associate the help text with the input for screen readers.

Reply:
in this case why not have the id of the help text as `id={describedBy}`?

Reply:
> in this case why not have the id of the help text as `id={describedBy}`?

1. Because it's an optional param that might not be set.
2. There's no reason the outer component should need to set that since the help text is managed internally by the dropdown.